multiversx/mx-chain-go
# Fix import db on rc v1.6.0

## Reasoning behind the pull request
- import DB stopped abnormally on epoch 3
  
## Proposed changes
- due to the refactoring of the resolvers/requesters components, we need to call the SetEpochHandler on the storage header requester in order to make the import-db process run as designed. This call is necessary so the manual epochs handler component can open one epoch in advance as of the current processing epoch so the start of epochs events will work

## Testing procedure
- standard system test
- after the standard system test, execute an import-db process on all shards. Import-db should work.

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
